src/index.js

```diff
@@ -6,36 +6,83 @@ import Game from './Game'
 import { createStore } from 'redux'
 import './index.css';
 
-Mousetrap.bind('up', () => store.dispatch({type: 'ROTATE'}))
-Mousetrap.bind('left', () => store.dispatch({type: 'LEFT'}))
-Mousetrap.bind('right', () => store.dispatch({type: 'RIGHT'}))
+let pausedBoolean = false
 
-const reducer = ( state = new Game(), action ) => {
-  switch (action.type) {
+const dispatchTick = () => {
+  setTimeout( () => store.dispatch({ type: 'TICK' }), 400 )
+}
+
+const isGameInactive = ( state ) => {
+  return ( pausedBoolean === true ||
+    (state.game1.isGameOver() === true || state.game2.isGameOver() === true ) )
+}
+
+const gameStream = ( state = { 'game1': new Game(), 'game2': new Game() }, action ) => {
+  switch ( action.type ) {
     case 'TICK':
```

Comment:
These should be defined as constants (typically in the same place the actions are defined), and imported.  This avoid logic errors that may arise from typos in the string, since the constant would be guaranteed to be the same, and would yield an error if you had a type in the constant name (since you would not have created/imported it).